.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_TFT_Criteo_benchmarks.json

```diff
@@ -0,0 +1,156 @@
+        "h": 9,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "hiddenSeries": false,
+      "id": 2,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "8.1.8",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "alias": "cloudml_benchmark_cirteo_no_shuffle_10GB_runtime_sec",
+          "hide": false,
+          "query": "SELECT mean(\"value\") FROM \"cloudml_benchmark_cirteo_no_shuffle_10GB\" WHERE metric = 'runtime_sec' AND $timeFilter GROUP BY time($__interval),  \"metric\"",
+          "rawQuery": true,
+          "refId": "B"
+        },
+        {
+          "alias": "cloudml_benchmark_criteo_10GB_runtime_sec",
+          "hide": false,
+          "query": "SELECT mean(\"value\") FROM \"cloudml_benchmark_criteo_10GB\" WHERE metric = 'runtime_sec' AND $timeFilter GROUP BY time($__interval),  \"metric\"",
+          "rawQuery": true,
+          "refId": "A"
+        },
+        {
+          "alias": "cloudml_benchmark_criteo_fixed_workers_10GB_runtime_sec",
+          "hide": false,
+          "query": "SELECT mean(\"value\") FROM \"cloudml_benchmark_criteo_fixed_workers_10GB\" WHERE metric = 'runtime_sec' AND $timeFilter GROUP BY time($__interval),  \"metric\"",
+          "rawQuery": true,
+          "refId": "C"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "TFT Criteo benchmarks",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "transparent": true,
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "s",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    }
+  ],
+  "refresh": false,
+  "schemaVersion": 30,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "TFT Criteo Benchmarks",
+  "uid": "C9nj1oxVk",
```

Comment:
where does this come from?

Reply:
You can run  instructions mentioned at https://github.com/apache/beam/tree/master/.test-infra/metrics and then create a plot with appropriate metric name and query. After that is done and once you see the plot, you can save the plot metadata in json format.

Reply:
This metadata is created by grafana.